### ISSUE TITLE
There are two issues below，can you consider  on the agenda?

### ISSUE THREAD

**Issue by showdy** [Frosch2010/uCrop-n-Edit#1]
There are two issues below，can you consider  on the agenda?
1. Resolve overlay and cropImageview sliding conflicts
2. Support overlay rotation while cropImageView is also rotated
Specific effects refer to following video:

https://github.com/Frosch2010/uCrop-n-Edit/assets/13609650/98e557d1-2fc8-45f3-b6df-960aea6eba5a

**Comment by Frosch2010** [comment]
Yes, I will try my best. However I do not have much time in the next weeks to fix these issues.

Can you explain the first issues a bit more detailed, please?
